static/app/components/smartSearchBar/utils.tsx

```diff
@@ -181,37 +181,44 @@ export function generateOperatorEntryMap(tag: string) {
     [TermOperator.Default]: {
       type: ItemType.TAG_OPERATOR,
       value: ':',
-      desc: `${tag}:${t('[value] is equal to')}`,
+      desc: `${tag}:${t('[value]')}`,
+      documentation: 'is equal to',
```

Comment:
Is this just to match other operator descriptions which appear on the right?

I actually kinda like it more on the left, but it kind of depends on what typically goes on the right.

cc @vuluongj20

Reply:
One UX issue I can see I guess is on wide screens the right-aligned text is very far away, but if we put it on the left it should be greyed out instead of the same color like before.